code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -139,13 +139,14 @@ Contains:
 	flags_cover = HEADCOVERSEYES
 
 /obj/item/clothing/head/helmet/space/pirate/bandana
-	name = "pirate bandana"
+	name = "royal bandana"
+	desc = "A special bandana that is space proof as well as made with padded armor."
 	icon_state = "bandana"
 	item_state = "bandana"
 
 /obj/item/clothing/suit/space/pirate
-	name = "pirate coat"
-	desc = "Yarr."
+	name = "royal waistcoat "
+	desc = "A royal waistcoat made to be spaceproof as well as padded with armor."

```

Comment:
A royal, space-proof waistcoat. The inside of it is lined with reflective kevlar.